src/main/resources/edu/cornell/kfs/fp/cu-ojb-fp.xml

```diff
@@ -1101,5 +1101,37 @@
     <field-descriptor name="objectId" column="OBJ_ID" jdbc-type="VARCHAR" index="true"/>
     <field-descriptor name="noteText" column="ACH_NTE_TXT" jdbc-type="VARCHAR"/>
   </class-descriptor>
+  
+  <class-descriptor class="edu.cornell.kfs.fp.document.CorporateBilledCorporatePaidDocument" table="FP_PRCRMNT_DOC_T">
```

Comment:
Should the CBCP documents be mapped to a new table instead? Otherwise, the application could get confused about whether a given document is a PCard one or a CBCP one, unless you add some kind of discriminator column or other custom discriminator setup.